src/sentry/filters/web_crawlers.py

```diff
@@ -36,6 +36,8 @@
             r'spider[\/\s\)\;]',
             # Slack - see https://api.slack.com/robots
             r'Slack',
+            # Calypso AppCrawler
```

Comment:
tbh this comment is exactly the same as the string, so if it's not adding any additional value, I'd remove it. But I'd argue we should link to what this thing is.

Reply:
It's your call. I don't know anything about this. :)